pages/about.html

```diff
@@ -0,0 +1,108 @@
+            <h1 class="header-title">United Nations<br> Climate Change<br> Conference 2022</h1>
+            <p class="header-msg about-headline">COP 27 will focus on developing greater resilience and reducing vulnerability to climate change. Finance relates to the $100 billion pledge and the importance of showing progress on delivering this.
+                In particular, it will be a key component of helping to strengthen resilience and achieving these goals.
+            </p>
+            <address class="date-with-location">
+                <p class="date about-address">Please contact us for more information about Conference of the Parties (COP27)</p>
+                <a class="location about-address"><strong>secretariat@unfccc.int</strong></a>
+            </address>
+        </div>
+    </header>
+    <section id="program" class="main-program about-program">
+        <div class="about-display">
+            <h2 class="main-program-header">UN Climate Change Conference Logo</h2>
+            <hr class="red-line">
+            <p class="logo-info mobile-logo-info">The logo of the UN Climate Change Conference represents the global action to make our world safe</p>
+            <p class="logo-info desktop-logo-info">
+                The logo of the UN Climate Change Conference represents the global action to make our world safe.
+                The United Nations Climate Change Conference are yearly conferences held in the framework of the UNFCCC.
+                They serve as the formal meeting of the UNFCCC Parties (Conferences of the Parties) (COP) to assess progress in dealing with climate change, and beginning in the mid-1990s,
+                to negotiate the Kyoto Protocol to establish legally binding obligations for developed countries to reduce their greenhouse gas emissions.
+                This is the symbol that represents that.
+            </p>
+            <img src="../images/logo.png" alt="UN Climate Change Logo" class="about-logo">
+        </div>
+        <div class="logo-hr">
+            <hr>
+        </div>
+    </section>
+    <section class="previous-conferences">
+        <h2 class="previous-conferences-header">Previous Conferences</h2>
+        <hr class="red-line">
+        <p class="previous-conferences-msg">Below are some of the previous conferences that have taken place in the past.</p>
+        <div class="cop">
+            <div class="conference cop26">
+                <h3 class="conference-header">2021</h3>
+                <p class="conference-msg">The 2021 United Nations Climate Change Conference (COP26) in Glasgow</p>
+            </div>
+            <div class="conference-overlay"></div>
+            <div class="conference cop24">
+                <h3 class="conference-header">2018</h3>
+                <p class="conference-msg">The 2018 United Nations Climate Change Conference (COP24) in Katowice</p>
+                <div class="conference-overlay"></div>
+            </div>
```

Comment:
- Please try to replicate the exact design specifications. This will give confidence to recruiters in your ability to replicate Figma designs. The images have a linear gradient., please try to fix this in your project
Your project:
![previous conf project](https://user-images.githubusercontent.com/73607512/190425444-71cfc888-c39c-49b5-a079-01094c63d521.PNG)
Design Template:
![previous conf design template](https://user-images.githubusercontent.com/73607512/190425500-65d3f75b-a71a-48a1-8f56-eb19cfd0c78b.PNG)